cves/kernel/CVE-2021-38198.yml

```diff
@@ -344,8 +349,9 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+        There were two signed-off by fields in the commit message
```

Comment:
Might be beneficial in supplying names of people who have signed off on these commit messages